spec/acceptance/ios_radius_server_group_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'spec_helper_acceptance'
+
+describe 'ios_radius_server_group' do
+  it 'add ios_radius_server_group' do
+    pp = <<-EOS
+    ios_radius_server_group { "bill":
```

Comment:
this is the same group name used in `radius_server_group`. To be sure that one does not influence the other can you change the name to something unique.

Reply:
Changed to ted